plugins/git/gitlab_statistics

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python3
+# -*- python -*-
+# -*- coding: utf-8 -*-
+#
+# Plugin to monitor Gitlab status
+#
+# Usage: Place in /etc/munin/plugins/ (or link it there using ln -s)
+#   Add this to your /etc/munin/plugin-conf.d/munin-node:
+#       [gitlab]
+#       user nobody
+#       logarithmic 1
+#       hostname gitlab.example.com # required
+#       token YourPrivateTokenHere # required
+#
+# 2019-10-02: v 1.0 pcy <pcy.ulyssis.org>: created
+#
+# Parameters understood:
+#
+#   config   (required)
+#   autoconf (optional - used by munin-config)
+#
+# magic markers:
+#%# family=auto
+#%# capabilities=autoconf
+
+import os
+import requests
```

Comment:
`requests` is not part of the standard library. Thus you would need the following details to use it properly:
* use a conditional import
* `autoconf` responds with `no (...)` if it is missing
* add the requirement to the documentation header

Personally I use `requests` only for interactive http retrieval (since it is easy to use), but I would refrain from using it in code due to its non-deterministic behaviour (e.g. [#4842](https://github.com/psf/requests/issues/4842)).

Instead you may want to use `urllib` and `json` (core libraries):
```python
raw_data = urllib.request.urlopen(url)
content = json.loads(raw_data) 
```

Reply:
Done.